src/processes/scoring/update-user-score-on-daily-process-phase1.js

```diff
@@ -54,6 +54,10 @@ const updateUserScoreOnDailyProcessPhase1 = async (
     userDoc.last_daily_process = processTime;
     userDoc.updated_at = moment().utc().format(REGULAR_TIME_FORMAT);
 
+    // set user anonymous
+    const currentUser = await UsersFunction.getUserFollowerList(userDoc._id);
+    userDoc.is_anonymous = currentUser.is_anonymous;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getUserFollowerList` function name suggests that it retrieves a list of followers for a user. However, it's being used here to determine if the user is anonymous or not. This could lead to confusion and potential misuse in the future. Consider renaming the function to better reflect its purpose or create a new function specifically for checking the anonymity of a user.

```diff
- const currentUser = await UsersFunction.getUserFollowerList(userDoc._id);
+ const currentUser = await UsersFunction.getUserAnonymityStatus(userDoc._id);
  userDoc.is_anonymous = currentUser.is_anonymous;
```

Also, there's no error handling for the async operation. If `getUserFollowerList` fails for any reason (network issues, server errors, etc.), the code will throw an unhandled promise rejection. It's recommended to wrap this in a try-catch block to handle potential errors.

```diff
- const currentUser = await UsersFunction.getUserFollowerList(userDoc._id);
+ let currentUser;
+ try {
+   currentUser = await UsersFunction.getUserFollowerList(userDoc._id);
+ } catch (error) {
+   console.error(`Failed to get user follower list: ${error}`);
+   // Handle error appropriately, e.g., retry, return, throw, etc.
+ }
  userDoc.is_anonymous = currentUser.is_anonymous;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->